cmd/kube-apiserver/app/server_test.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"regexp"
+	"testing"
+)
+
+func TestLongRunningRequestRegexp(t *testing.T) {
+	regexp := regexp.MustCompile(defaultLongRunningRequestRE)
+	dontMatch := []string{
+		"/api/v1/watch-namespace/",
+		"/api/v1/namespace-proxy/",
+		"api/v1/watch/stuff",
+		". anything",
+		"/ that",
+	}
+	doMatch := []string{
+		"/api/v1/pods/watch",
+		"/api/v1/default/service/proxy",
+		"/api/v1/pods/proxy/path/to/thing",
```

Comment:
I think we should also match these urls, which are expected to be long-running:
* `/api/v1/namespaces/myns/pods/mypod/logs`
* `/api/v1/namespaces/myns/pods/mypod/portforward`
* `/api/v1/namespaces/myns/pods/mypod/exec`
* `/api/v1/watch/namespaces/myns/pods`